relay-server/src/actors/global_config.rs

```diff
@@ -85,6 +85,9 @@ impl UpstreamQuery for GetGlobalConfig {
 pub struct Get;
 
 /// The message for receiving a watch that subscribes to the [`GlobalConfigService`].
+///
+/// The global config service must be up and running, else the subscription
+/// fails.
```

Comment:
What happens exactly when you try to subscribe earlier? Might be worth elaborating.